app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationDialogFragment.java

```diff
@@ -0,0 +1,115 @@
+package org.mozilla.focus.biometrics;
+
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.arch.lifecycle.LifecycleObserver;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.RequiresApi;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatDialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import org.mozilla.focus.R;
+
+@RequiresApi(api = Build.VERSION_CODES.M)
+public class BiometricAuthenticationDialogFragment extends AppCompatDialogFragment implements LifecycleObserver {
+    public static final String FRAGMENT_TAG = "biometric-dialog-fragment";
+    private TextView mErrorTextView;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Do not create a new Fragment when the Activity is re-created such as orientation changes.
+        setRetainInstance(true);
+        setStyle(DialogFragment.STYLE_NORMAL, android.R.style.Theme_Material_Light_Dialog);
+    }
+
+    @Override
+    public void onCancel(DialogInterface dialog) {
+        super.onCancel(dialog);
+        Intent startMain = new Intent(Intent.ACTION_MAIN);
+        startMain.addCategory(Intent.CATEGORY_HOME);
+        startMain.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+        startActivity(startMain);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
```

Comment:
\[PMD\] INFO: The method 'onCreateView\(LayoutInflater, ViewGroup, Bundle\)' has a NCSS line count of 18\.

Reply:
Confused by this one, I broke the line up and still getting this error.